package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@flatfile/adapter",
-  "version": "2.5.0",
+  "version": "2.6.0",
```

Comment:
We are using @flatfile/react@0.1.8 . this internal package was upgraded with 0.1.8 version of @flatfile/react as well and it caused our application build fail for our production deployments on new customers. IMO if internal packages are being upgraded then it should go with new version of parent package which is being used by clients to use flatfile.

Reply:
@MaksymBlank and @NateFerrero FYI